src/main/java/hudson/remoting/Request.java

```diff
@@ -233,6 +233,8 @@ public boolean isDone() {
 
             public RSP get() throws InterruptedException, ExecutionException {
                 synchronized(Request.this) {
+                    String oldThreadName = Thread.currentThread().getName();
```

Comment:
This is the same variable name used everywhere. I think it's `old (thread name)` not `(old thread) name`